### PR TITLE
Add tutorials page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Created a static tutorials page.

## QA
- Go the demo
- Click on Tutorials in the navigation
- Check the pages looks ok
- Check it contains the 5 juju tutorials from https://tutorials.ubuntu.com/ if you search "juju"
- Check the tutorials load correctly

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/966

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "license": "LGPL v3",
   "scripts": {
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
+    "watch": "watch -p 'static/sass/*.scss' -c 'yarn run build'",
```
Comment (hatched) [inline package.json]: I'm curious as to why you removed the recursive watcher?

Comment (anthonydillon) [reply package.json]: This only watches files in the root scss directory. The main styles should be contained in a patterns  directory but in this case it's a custom directory.

Comment (hatched) [reply package.json]: I'm a tiny bit concerned that someone may add a sub dir to that sass dir and it won't properly build and won't know why.

If there are no sub directories then keeping the `**/*` doesn't have any negative effects, does it?

**Comment by clagom** [comment]
@barrymcgee was this reviewed?

File: package.json (hunk 1 above)
Comment (anthonydillon) [reply package.json]: Fair point, I've reverted for now.

**Comment by hatched** [comment]
Looks like this one has a lot of conflicts now :)

**Comment by anthonydillon** [comment]
@hatched conflicts resolved

**Comment by hatched** [comment]
Looks good, thanks @anthonydillon 
The tutorials need images though :) Sometime in the future...

**Comment by cristinadresch** [comment]
@clagom

**Pull request merged**